homeassistant/components/amberelectric/sensor.py

```diff
@@ -122,8 +127,8 @@ def native_value(self) -> str | None:
         interval = intervals[0]
 
         if interval.channel_type == ChannelType.FEED_IN:
-            return round(interval.per_kwh, 0) / 100 * -1
-        return round(interval.per_kwh, 0) / 100
+            return format_cents_to_dollars(interval.per_kwh) * -1
```

Comment:
Just noticed this turns the pricing negative, I've asked quickly, but the energy dashboard isn't compatible with that.

Reply:
That's because the API returns the price as negative (it's a ledger) - so this flips it back around to be compatible with the energy dashboard.

Reply:
And it does support negatives - I had a negative feed in the day yesterday (I owe money for the solar I generated), and it worked

![image](https://user-images.githubusercontent.com/19585/136105458-54665992-cdae-4969-861c-44acbc6fbcdd.png)

![image](https://user-images.githubusercontent.com/19585/136105479-59e57b5e-985e-4a11-99a3-6b66dbc41904.png)

(Ignore the slight discrepancy - there is a timing issue that causes things to be out by a few cents over the day, generally)